plugins/ImageGraph/ImageGraph.php

```diff
@@ -56,7 +56,7 @@ public function getReportMetadata(&$reports, $info)
         $idSite = $info['idSite'];
 
         // If only one website is selected, we add the Graph URL
-        if (empty($idSite)) {
+        if (empty($idSite) || 'all' === $idSite) {
```

Comment:
Not super important but we could probably make these checks stronger w/ `!is_numeric($idSite)`. Think it would be a good idea?

Reply:
changed that